lib/command-line-parse.js

```diff
@@ -0,0 +1,15 @@
+const argv = require('minimist')(process.argv.slice(2))
+
+function ParsedCmdLine(){
+    self = this
+    self.sheets = argv._
+    self.filename = argv.f
+    self.sheetsDir = argv.D ? argv.D : ""
+    self.all = argv.a
+    const validate = ()=>{
+        return self.filename
```

Comment:
Please validate other arguments as well. Both conversions needs XLS file and the JSON directory as bare minimum.